apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -1869,8 +1869,8 @@
       "targetAccount": "Target Account"
     },
     "sell": {
-      "notice": "Verify the Sell details on your device before sending it. The addresses are exchanged securely so you don’t have to verify them.",
-      "confirm": "Confirm Sell transaction"
+      "notice": "Verify the details on your device before sending it. The addresses are exchanged securely so you don’t have to verify them.",
+      "confirm": "Confirm transaction"
```

Comment:
`src/renderer/components/DeviceAction/rendering.tsx` is using this constants, please create your own constants.